types/unused-webpack-plugin/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for unused-webpack-plugin 2.4
+// Project: https://github.com/MatthieuLemoine/unused-webpack-plugin#readme
+// Definitions by: Remco Haszing <https://github.com/remcohaszing>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import { WebpackPluginInstance, Compiler } from 'webpack';
+
+export = UnusedWebpackPlugin;
+
+declare namespace UnusedWebpackPlugin {
+    interface UnusedWebpackPluginOptions {
+        /**
+         * array of directories where to look for unused source files.
+         */
+        directories: string[];
+
+        /**
+         * array of exclude patterns when looking for unused source files.
+         */
+        exclude?: string[];
```

Comment:
the default value is an empty array
https://github.com/MatthieuLemoine/unused-webpack-plugin/blob/master/index.js#L7

Reply:
You’re right. I feel like most DefinitelyTyped reviewers don’t pay any attention to the docs. I appreciate that you do so. :smile:

Reply:
Agreed, thank you for helping out @martin-badin!